pydy/viz/tests/test_visualization_frame_scene.py

```diff
@@ -8,11 +8,11 @@
 from numpy.testing import assert_allclose
 
 # local
-from ..shapes import Cylinder
-from ..visualization_frame import VisualizationFrame
-from ..camera import PerspectiveCamera, OrthoGraphicCamera
-from ..scene import Scene
-from ..light import PointLight
+from pydy.viz.shapes import Cylinder
```

Comment:
Do these have to change because of Python 3? If not, I'd prefer to leave them as relative imports.